website/docs/reference/segments.mdx

```diff
@@ -38,13 +38,19 @@ Segments are collections of strategy constraints. To satisfy a segment, _all_ th
 
 If an activation strategy has a segment _and_ additional constraints applied, the segment _and_ the strategies must all be satisfied. Similarly, if an activation strategy has multiple segments, then they must _must all be satisfied_.
 
-## Segment limits
+## Segment limits (updated in v5.1.0)
```

Comment:
I don't think the "updated in X" belongs in the heading here. Makes it feel like an announcement  / blog post

```suggestion
## Segment limits
```